components/match2/wikis/dota2/matchgroup_input_custom.lua

```diff
@@ -416,7 +416,7 @@ function matchFunctions.getOpponents(match)
 
 			-- Retrieve icon for team
 			if opponent.type == Opponent.team then
-				opponent.icon = opponentFunctions.getIcon(opponent.template)
+				opponent.icon, opponent.icondark = opponentFunctions.getIcon(opponent.template)
```

Comment:
What happens in lua when this function returns null?

Reply:
it just doesn't set the icon nor icondark value in the opponent
![Screenshot 2022-03-07 11 03 20](https://user-images.githubusercontent.com/75081997/157013519-f4014756-a3c5-4487-a363-a6c37b44858d.png)
![Screenshot 2022-03-07 11 03 23](https://user-images.githubusercontent.com/75081997/157013528-11ee09ae-01b9-4cd6-8adf-814cb247e65f.png)

Reply:
So they both become null. Interesting.

Reply:
in lua you theoretically can use stuff like
```lua
function p.test()
	local a, b, c
	a, b, c = 'a', 'b'
	mw.logObject({a = a, b = b, c = c})
end
```
and it returns
```
table#1 {
  ["a"] = "a",
  ["b"] = "b",
}
```